Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -47,13 +37,30 @@ public static bool CheckMixedRealityConfigured(bool renderEditorElements = true,
                 {
                     EditorGUILayout.HelpBox("No Active Profile set on the Mixed Reality Toolkit.", MessageType.Error);
                 }
-
                 return false;
             }
 
             return true;
         }
 
+        /// <summary>
+        /// If MRTK is not initialized in scene, adds & initializes instance to current scene
```

Comment:
>& [](start = 54, length = 1)

I think this character might throw off the docs build (I remember the docs build failing on this one:

[https://dev.azure.com/aipmr/MixedRealityToolkit-Unity-CI/_build/results?buildId=1633](https://dev.azure.com/aipmr/MixedRealityToolkit-Unity-CI/_build/results?buildId=1633))

I think that just making it "and" should be enough to get it good.